site/composer.lock

```diff
@@ -4819,20 +4819,20 @@
         },
         {
             "name": "symfony/polyfill-php80",
```

Comment:
The lock file still has a polyfill object here. Is this expected, or should this reference be removed?

Reply:
Searching for `symfony/polyfill-php80` in `composer.lock`, can see that it's under the `require` block for `doctrine/orm` and `symfony/polyfill-php83` (which is in turn required by `symfony/http-foundation`).

So while it's no longer a direct dependency in `package.json`, it's a transitive dependencies, and so will remain in the lock file as it lists out all direct and transitive dependencies of the project.